src/v2/Apps/Partner/PartnerApp.tsx

```diff
@@ -40,7 +42,7 @@ export const PartnerApp: React.FC<PartnerAppProps> = ({
               <Separator />
             </FullBleed>
 
-            <NavigationTabs partner={partner} />
+            {!isNonSubscriber && <NavigationTabs partner={partner} />}
```

Comment:
The `!isNonSubscriber` syntax here is really puzzling to me. This should be `isSubscriber` logically:)

But even on a higher level - if I understand correctly - we try to encapsulate if this should be a `fullProfile` page vs `limitedDisplay` page - right?

Maybe we can have something along those lines - `fullProfileEligeable`? , `fullProfileEnabled`? , `limitedProfile` if we want to go negative route here?

And I guess along the line this page is also will be tweaked to support Auction Partners?  Would they have full profile?

So I guess my suggestion would be to try to think of a good medium term state that we want to get to and see if our naming could reflect that. `Subscriber` term conflates what we are trying to represent here in my mind. Right now the check happen to be for subscription state but ultimately what we are trying to represent is a UI concept of what should be displayed and since we are just adding this name - I suggest our name should be a bit more descriptive of what we are trying to control in UI.

Reply:
> we try to encapsulate if this should be a fullProfile page vs limitedDisplay page - right?

Yes, you are right

> And I guess along the line this page is also will be tweaked to support Auction Partners? Would they have full profile?

I'm not sure, because I have no information about Auction Partners. 

I'm okay with `fullProfileEligeable`, also agree that it will be more descriptive then current.

Reply:
Rewrote code to use fullProfileEligible instead of isNonSubscriber. Please take a look.